src/index.js

```diff
@@ -34,6 +34,24 @@ function parsePart(part) {
   });
 }
 
+function validatePart(timeElementValues, validator){
+  let elements = [], isValid = true;
+
+  if(timeElementValues != null) {
```

Comment:
If `timeElementValues` is `null` or `undefined`, that should also cause an error.

I am also okay if we don't check this at all and assume it is an array of `{ start, end }` objects. That should be safe since the code below will throw an error anyway.

Reply:
I think timeElementValues are null when the string contains a * for any timeElement 
(say: 9 * 23 11 1-5, here the * gets converted to null ). Error shouldn't be thrown in this scenario, because it is still a valid time range